punjabi/punjabi.md

```diff
@@ -0,0 +1,2 @@
+Click [here](ajit/ajitnews.md) for punjabi news
+Click [here](pollywood/movies.md) for Punjabi movies
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.